app.py

```diff
@@ -3,57 +3,62 @@
 import os
 import requests
 
-app = Flask(__name__)
-app.config.from_object(os.environ['APP_SETTINGS'])
-
-app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
-db = SQLAlchemy(app)
-
-from models import CelestialBodies, Landmark, User, Passenger
-
-@app.route('/')
-def hello():
-    return "App is running"
-
-@app.route('/api/v1/celestial_bodies', methods = ['GET'])
-def get_bodies():
-    try:
-        bodies = CelestialBodies.query.all()
-        return jsonify({'data': [e.serialize() for e in bodies]})
-    except Exception as e:
-        return(str(e))
-
-@app.route('/api/v1/celestial_bodies/<id>', methods = ['GET'])
-def get_one_body(id):
-    try:
-        celestial_body = CelestialBodies.query.get(id)
-        return jsonify(celestial_body.serialize())
-    except Exception as e:
-        return(str(e))
-
-@app.route('/api/v1/news', methods = ['GET'])
-def get_api():
-    try:
-        response = requests.get('https://spaceflightnewsapi.net/api/v1/articles')
-        return response.json()
-    except Exception as response:
-        return(str("Bad Request"))
-
-@app.route('/api/v1/celestial_bodies/<id>/landmarks', methods = ['GET'])
-def get_landmarks(id):
-    try:
-        landmarks = Landmark.query.filter(Landmark.celestial_body_id == id)
-        return jsonify({'data': [e.serialize() for e in landmarks]})
-    except Exception as e:
-        return(str(e))
-
-@app.route('/api/v1/landmarks/<id>', methods = ['GET'])
-def get_landmark(id):
-    try:
-        landmark = Landmark.query.get(id)
-        return jsonify(landmark.serialize())
-    except Exception as e:
-        return(str(e))
+db = SQLAlchemy()
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(os.environ['APP_SETTINGS'])
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    db.init_app(app)
+    from main import main
+    app.register_blueprint(main)
+    return app
```

Comment:
create_app gives us a way to instantiate the app for testing. When we register the blueprint, this allows us to use the routes for our endpoints. Since this is now inside the create app method, it will give any instance of the app that uses this method for creation to use the endpoints.